--- ./batik-dom-svg-MANIFEST.MF	2011-07-08 13:40:27.911556763 -0400
+++ ./batik-dom-svg-MANIFEST.MF	2011-07-08 13:41:33.834333213 -0400
@@ -8,8 +8,7 @@
  g.apache.batik.dom;bundle-version="[1.6.0,1.7.0)",org.apache.batik.pa
  rser;bundle-version="[1.6.0,1.7.0)",org.apache.batik.util;bundle-vers
  ion="[1.6.0,1.7.0)",org.w3c.css.sac;bundle-version="[1.3.0,1.4.0)",or
- g.w3c.dom.smil;bundle-version="[1.0.0,1.1.0)",org.w3c.dom.svg;bundle-
- version="[1.1.0,1.3.0)"
+ g.w3c.dom.svg;bundle-version="[1.1.0,1.3.0)"
 Bundle-Version: 1.6.0.v201011041432
 Export-Package: org.apache.batik.dom.svg;version="1.6.0",org.apache.ba
  tik.dom.svg12;version="1.6.0"
